src/v2/Apps/Artist/Components/ArtistTopWorksRail/ArtistTopWorksRail.tsx

```diff
@@ -0,0 +1,150 @@
+import { Box, Flex, Sans } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistTopWorksRail_topRailArtworks } from "v2/__generated__/ArtistTopWorksRail_topRailArtworks.graphql"
+import { Carousel } from "v2/Components/Carousel"
+import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
+import { Media } from "v2/Utils/Responsive"
+import { StyledLink } from "v2/Apps/Artist/Components/StyledLink"
+
+interface ArtistTopWorksRailProps {
+  topRailArtworks: ArtistTopWorksRail_topRailArtworks
+  onOverviewTab?: boolean
+}
+
+export const ArtistTopWorksRail: React.FC<ArtistTopWorksRailProps> = ({
+  topRailArtworks,
+  onOverviewTab,
+}) => {
+  const carouselHeight = 300
+  const artworks = topRailArtworks?.images?.edges ?? []
+  const handleViewWorksClick = event => {
+    return setTimeout(() => scrollToArtworkView(event), 500)
+  }
+
+  const scrollToArtworkView = event => {
+    const $element = document.getElementById("jump--artworkFilter")
+
+    if ($element) {
+      const { top } = getElementPosition($element)
+      window.scrollTo({
+        top: top,
+      })
+    }
+  }
```

Comment:
Rather than going this directly can use the `ScrollIntoView` component? Here's an [example implementation](https://github.com/artsy/force/blob/77327bbda32a5364300f63ef686af198f7d0342d/src/v2/Components/Pagination.tsx#L36), though that might need to be adjusted sligtly via a new prop to add some smooth scroll behavior.

Reply:
I wasn't able to get it working so I extracted the scrollIntoView function into a helper method used both by the `<ScrollIntoView />` comp + this rail!

Reply:
That works 👍